.github/workflows/release.yml

```diff
@@ -58,3 +58,16 @@ jobs:
         if: ${{ steps.release.outputs.release_created }}
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+  homebrew:
+    name: "Bump Homebrew formula"
+    runs-on: ubuntu-latest
+    needs: release
+    steps:
+      - uses: mislav/bump-homebrew-formula-action@b3327118b2153c82da63fd9cbf58942146ee99f0 # v3
```

Comment:
Did you test the action on a branch with the release? Looking at the docs, I think we should set `tag-name` with the output of the release.